src/common-utils/main.sh

```diff
@@ -564,7 +564,7 @@ chmod +rx /usr/local/bin/code
 
 # systemctl shim for Debian/Ubuntu - tells people to use 'service' if systemd is not running
 if [ "${ADJUSTED_ID}" = "debian" ]; then
-    cp -f "${FEATURE_DIR}/bin/systemctl" /usr/local/bin/systemctl
+    cp -fL "${FEATURE_DIR}/bin/systemctl" /usr/local/bin/systemctl
```

Comment:
In the tests you can manually create a symlink to some other folder (eg. cp -f "/tmp/testl-systemctl" /usr/local/bin/systemctl)

Then run `cp -fL "${FEATURE_DIR}/bin/systemctl" /usr/local/bin/systemctl` and ensure that the symlink worked and it now points to ${FEATURE_DIR}/bin/systemctl instead of /tmp/testl-systemctl

Reply:
Hi @samruddhikhandale ,
I have tested this scenario in one of the container of common-utils. followed following steps:

1. created a file with a dummy content "Hi". (/tmp/testl-systemctl)
2. used this command: `cp -f /tmp/testl-systemctl /usr/local/bin/testl-systemctl`
3. checked the destination file is forcefully copied. content of the `/usr/local/bin/testl-systemctl` is same.
4. Now, used this command: `cp -fL /usr/local/bin/systemctl /usr/local/bin/testl-systemctl` 
5. checked the destination file is now pointing to `/usr/local/bin/systemctl`
![image](https://github.com/devcontainers/features/assets/159782310/9c128c43-db88-4f6f-bf35-aafc7dd0dce6)

Is this what you are looking for?

Reply:
This looks alright, thanks!